frontend/spree_frontend.gemspec

```diff
@@ -23,7 +23,6 @@ Gem::Specification.new do |s|
   s.add_dependency 'spree_core', s.version
 
   s.add_dependency 'bootstrap',       '>= 4.3.1', '< 4.6.0'
-  s.add_dependency 'glyphicons',      '~> 1.0.2'
```

Comment:
@damianlegawiec Do you want this gem leaving in also? it's now unused in the frontend and backend?

Reply:
@MatthewKennedy keep in mind there might be several spree extensions which still depend on some glyphicons (e.g. https://github.com/spree-contrib/spree_multi_vendor/blob/84462d7ad4e8fba3d4e15aae71c3c7f7d48f3ea6/app/overrides/spree/admin/shared/_main_menu.rb#L14)